news/6823.bugfix.rst

```diff
@@ -0,0 +1 @@
+This change will preserve path in it's original letter-case during displaying.
```

Comment:
```suggestion
This change will preserve path in its original letter-case during displaying.
```

<old-man-yelling-at-clouds> I don’t unsderstand how kids these days can consistently spell it’s and its wrong. </old-man-yelling-at-clouds>

Reply:
No problem, will be fixed in next revision.